demos/notebooks/demo_pipeline.ipynb

```diff
@@ -453,16 +683,54 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#%% PLOT COMPONENTS\n",
-    "cnm2.estimates.plot_contours_nb(img=Cn, idx=cnm2.estimates.idx_components)"
+    "cnmf_model = cnmf.CNMF(n_processes, \n",
+    "                       params=parameters, \n",
+    "                       dview=multiprocessing_pool)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once initialized, we we could immediately run `cnmf_model.fit(images)` if we knew the parameters were good -- the parameters in your CNMF estimator object are accessable in `cnmf_model.params`. However, before running the algorithm, let's discuss the more critical parameters that you will most likely need to tweak when running CNMF on your own data. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Key parameters for CNMF\n",
+    "\n",
+    "`rf (int)`: *patch half-width*\n",
+    "\n",
+    "> `rf`, which stands for 'receptive field', is the half width of patches in pixels. The full patch width is `2*rf + 1`. `rf` should be *at least* 3-4 times larger than the observed neuron diameter. This is so that the CNMF algorithm will be able to properly estimate local background activity. Also the larger the patch size, the less parallelization will be used by Caiman. If `rf` is set to `None`, then no parallelization will be used and CNMF will be run on the entire field of view. \n",
```

Comment:
There are many words in this section that can be removed and other rephrasings that won't change meaning but will reduce bulk, making this less of a wall of text.

Reply:
Cleaned up some.